theia-blueprint-updater/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
-  "name": "theia-example-updater",
+  "name": "theia-blueprint-updater",
   "version": "1.9.0",
-  "description": "Theia - Updater and branding for the installer",
+  "description": "Eclipse Theia blueprint product updater",
```

Comment:
Propose just "Eclipse Theia Blueprint updater"

Reply:
Went for Eclipse Theia Blueprint Updater and Customization